COLLADA2GLTF/extensions/o3dgc-compression/GLTF-Open3DGC.cpp

```diff
@@ -326,8 +326,8 @@ namespace GLTF
             
             for (size_t j = 0 ; j < attributesCount ; j++) {
                 shared_ptr <GLTFAccessor> meshAttribute = mesh->getMeshAttribute(semantic, j);
-                vertexCount = meshAttribute->getCount();
-                size_t componentsPerElement = meshAttribute->componentsPerElement();
+                vertexCount = (long)meshAttribute->getCount();
```

Comment:
It actually will give a warning. In Visual Studio `size_t` is actually an `unsigned __int64` (8 bytes) and `long` is the same size as an `int` (4 bytes). `long`s only requirement that they must at least be the same size as an `int`.